.github/workflows/rust.yml

```diff
@@ -0,0 +1,15 @@
+name: Rust
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build
+      run: cargo build --verbose
```

Comment:
mmm seems you didn't need to setup Rust in the CI lane; then it must mean that GithubActions VMs already include it; but how can we make sure the version used is the stable one?

Reply:
I am not sure, I will check how to make sure the version used is the stable one.

Reply:
let's print version number in the console at least

Reply:
> let's print version number in the console at least

Done. Seems that it is Rust 1.41.1 and Cargo 1.41.0.

Reply:
I modified rust.yml to setup the rust to stable version.